UnitTestProject/TrainerTest.cs

```diff
@@ -5,6 +5,133 @@
 
 namespace Tests
 {
+    [TestClass] //ToDo: Move Test into a PlayerTest.cs
+    public class PlayerTest
```

Comment:
This commit is reverting changes backwards. It was fixed by you, then when you did your merge, it went backwards. Please cherry this out.